### ISSUE TITLE
"how to add snippets" section

### ISSUE THREAD

**Issue by oyes77** [oleeskild/obsidian-digital-garden#68]
is there a way to add snippets to your page? i don't know if un custom-css you can just copypaste and they work, maybe making it a told feature could help someone, and if it's not possible, then adding it as a feature would be cool

**Comment by oleeskild** [comment]
There is no "feature" for this, but as you have the digital garden template as a repo on your github, you can modify it however you see fit. 
So you could for example edit the src/site/_includes/layouts/note.njk file and add whatever snippets you want.

You can also edit the src/site/styles/custom-style.scss file and add any custom css there. 

Would that solve your issue?

**Comment by oleeskild** [comment]
Re-reading the title of the issue, am I correct in interpreting the issue in that you would like this to be documented better somewhere?

**Comment by oyes77** [comment]
> Re-reading the title of the issue, am I correct in interpreting the issue in that you would like this to be documented better somewhere?

yes, it would be great to add in the readme, like the how to edit css theme, make it clear if that is the same with snippets or not, like you just explained me

**Comment by kianenigma** [comment]
I just learned about Obsidian's default way to [inject css snippets](https://help.obsidian.md/Extending+Obsidian/CSS+snippets), and it is quite cool.

But, it is unfortunate that DG won't pick them up automatically. I actually think this can be automated: 

1. DG would look at `vault/snippets/*.css`
2. Filter by those that are active in global settings
3. Auto inject them into `src/site/styles/custom-style.scss`
4. Given this, I don't see a reason to let the content css class to be configurable, it should use Obsidian's default, [which is `cssclasses`](https://help.obsidian.md/Editing+and+formatting/Properties)
